lib/database/repositories/QcFlagRepository.js

```diff
@@ -242,14 +242,20 @@ class QcFlagRepository extends Repository {
             mcReproducible,
             flagsList,
             verifiedFlagsList,
-        }) => ({
-            runNumber,
-            bad,
-            effectiveRunCoverage: parseFloat(effectiveRunCoverage) || null,
-            mcReproducible: Boolean(mcReproducible),
-            flagsIds: [...new Set(flagsList.split(','))],
-            verifiedFlagsIds: verifiedFlagsList ? [...new Set(verifiedFlagsList.split(','))] : [],
-        }));
+        }) => {
+            if ((effectiveRunCoverage ?? null) != null) {
+                effectiveRunCoverage = Math.min(1, Math.max(0, parseFloat(effectiveRunCoverage)));
+            }
```

Comment:
Is that min-max necessary, as timestamps are fixed by the migration?

Reply:
Yes unfortunately it is. Some timestamps are fixed, but not all of them, and we still have a few flags outside run that causes negative summaries.